encode-decode/decodeHTMLEntitities.js

```diff
@@ -0,0 +1,292 @@
+# @author Timothée Ruffenach
+# Version 1.0
+# decode HTML Entities
```

Comment:
This is not valid comment for JavaScript (there are others that need to be corrected).